packages/algob/src/types.ts

```diff
@@ -363,7 +377,7 @@ export interface AlgobDeployer {
   putMetadata: (key: string, value: string) => void
   getMetadata: (key: string) => string | undefined
   deployASA: (name: string, flags: ASADeploymentFlags) => Promise<ASAInfo>
-  deployASC: (name: string, source: string, account: Account) => Promise<ASCInfo>
+  deployASC: (name: string, sc_params: Object, flags: ASCDeploymentFlags) => Promise<ASCInfo>
```

Comment:
How does `sc_params` differ from `flags`?
In spec there is only one object. You have to remove one parameter.

Reply:
scParams are the initialization parameters for contract that will be defined by user,

and flags are transaction parameters that are required to fund and makeLogic signature for contract.

Reply:
Let's then not have underscores

Reply:
ok